design_pim/guides/how_to_customize_menu.rst

```diff
@@ -19,14 +19,14 @@ To add an element at the root of the tree you can reuse the tab module provided
         config:
             title: pim_menu.item.import_profile        # You can define a translation key for the tab name
 
-After running the command ``bin/console pim:install:asset`` your new item should appear at the root of the menu
+After running the command ``rm -rf var/cache; bin/console pim:install:asset`` your new item should appear at the root of the menu.
```

Comment:
we should also run yarn webpack?

Reply:
Not if you modified only the yml conf. But I don't think you need to dump assets.